jwst/badpix_selfcal/badpix_selfcal_step.py

```diff
@@ -123,10 +123,12 @@ def process(self, input, selfcal_list=None, bkg_list=None):
             warnings.filterwarnings("ignore", category=RuntimeWarning, message="All-NaN")
             minimg = np.nanmin(np.asarray(selfcal_3d), axis=0)
         bad_indices = badpix_selfcal.badpix_selfcal(minimg, self.flagfrac_lower, self.flagfrac_upper, self.kernel_size, dispaxis)
-
+        
+        
         # apply the flags to the science data
         input_sci = badpix_selfcal.apply_flags(input_sci, bad_indices)
 
+        self.log.info(f"Number of new bad pixels flagged: " {len(bad_indices[0])}")
```

Comment:
```suggestion
        self.log.info(f"Number of new bad pixels flagged: {len(bad_indices[0])}")
```

Reply:
looks like there's a syntax error here, helpfully caught by our CI

Reply:
@emolter I did not see you labeled it no change log. I added one cause I thought it might need one.